fdbrpc/RangeMap.h

```diff
@@ -68,10 +68,20 @@ class RangeMap {
 	typedef MapPair<Key,Val> pair_type;
 public:
 	//Applications may decrement an iterator before ranges begin, or increment after ranges end, but once in this state cannot do further incrementing or decrementing
-	class Iterator {
+	template <bool isConst>
+	class IteratorImpl {
```

Comment:
This `IteratorImpl` class should be private within `RangeMap`.

Reply:
Done.